### PR TITLE
add sisyphus.js w/ npm autoupdate

### LINKED ISSUE #8069: [Request] Add sisyphus
**Library name:** Sisyphus
**Git repository url:** https://github.com/simsalabim/sisyphus
**npm package url(optional):** https://www.npmjs.com/package/sisyphus.js
**License(s):** https://github.com/simsalabim/sisyphus/blob/master/MIT-LICENSE
**Official homepage:** http://sisyphus-js.herokuapp.com

### PR DESCRIPTION
**Pull request by pvnr0082t**
PR for #8069 
@kennynaoh Please help me review this pull request, thank you.
Manually add the other versions from git repo because npm auto-update
can't get some of the old versions

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/simsalabim/sisyphus
 * Official website (optional, not the repository): http://sisyphus-js.herokuapp.com
 * NPM package url (optional): https://www.npmjs.com/package/sisyphus.js
 * GitHub / Bitbucket popularity (required):
   - Count of stars: 1795
   - Count of watchers: 54
   - Count of forks: 207
 * NPM download stats (optional):
   - Downloads in the last day: 0
   - Downloads in the last week: 2
   - Downloads in the last month: 14

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [ ] The lib has notable popularity
   * [x] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [ ] More than 500 downloads stats per month on npm registry
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

close #8069, cc @simsalabim

### CODE REVIEW

File: ajax/libs/sisyphus.js/package.json [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    "url": "https://github.com/simsalabim/sisyphus.git"
+  },
+  "keywords": [
+    "sisyphus",
+    "local-storage",
+    "localstorage"
+  ],
+  "author": "Alexander Kaupanin",
+  "license": "MIT",
+  "homepage": "http://sisyphus-js.herokuapp.com/",
```
Comment (kennynaoh) [inline ajax/libs/sisyphus.js/package.json]: Could this `https://sisyphus-js.herokuapp.com/` be better?
Thanks!

Comment (pvnr0082t) [inline ajax/libs/sisyphus.js/package.json]: Okay, I think it's better.

**Comment by PeterDaveHello** [comment]
The versions on GitHub look different with the version on npm, please confirm that.

**Comment by pvnr0082t** [comment]
@PeterDaveHello You mean the old versions don't be published on npm?

**Comment by PeterDaveHello** [comment]
@pvnr0082t I only see one version on npm, you will need to confrim that.

**Comment by simsalabim** [comment]
@PeterDaveHello @pvnr0082t Please see https://github.com/simsalabim/sisyphus/issues/142

Current version was updated to v1.1.3 on both GitHub and [npm](https://www.npmjs.com/package/sisyphus.js). Since `package.json` was introduced starting v1.1.2, precursory versions won't be published to npm, and there is a chance they will be removed from GitHub at some point.

Thanks for the work you're doing!

**Comment by pvnr0082t** [comment]
@kennynaoh @x09326 The latest version is v1.1.3. I add old versions due to npm auto-update
can't get some of the old versions. Please help me review this PR, thank you.

**Comment by x09326** [comment]
LGTM

**Comment by kennynaoh** [comment]
LGTM

**Comment by PeterDaveHello** [comment]
@kennynaoh you need to ping me since you are the second reviewer saying LGTM

**Comment by kennynaoh** [comment]
OK

**Pull request merged**